[2024-07-19T08:17:41.487+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crawling_dag_v2.read_s3_and_add_size_color manual__2024-07-19T08:13:53.191214+00:00 [queued]>
[2024-07-19T08:17:41.506+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crawling_dag_v2.read_s3_and_add_size_color manual__2024-07-19T08:13:53.191214+00:00 [queued]>
[2024-07-19T08:17:41.507+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-07-19T08:17:41.536+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): read_s3_and_add_size_color> on 2024-07-19 08:13:53.191214+00:00
[2024-07-19T08:17:41.549+0000] {standard_task_runner.py:60} INFO - Started process 1028 to run task
[2024-07-19T08:17:41.555+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crawling_dag_v2', 'read_s3_and_add_size_color', 'manual__2024-07-19T08:13:53.191214+00:00', '--job-id', '460', '--raw', '--subdir', 'DAGS_FOLDER/musinsa_dag.py', '--cfg-path', '/tmp/tmp5fzje06a']
[2024-07-19T08:17:41.560+0000] {standard_task_runner.py:88} INFO - Job 460: Subtask read_s3_and_add_size_color
[2024-07-19T08:17:41.703+0000] {task_command.py:423} INFO - Running <TaskInstance: crawling_dag_v2.read_s3_and_add_size_color manual__2024-07-19T08:13:53.191214+00:00 [running]> on host 9a455ef6c02c
[2024-07-19T08:17:41.899+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crawling_dag_v2' AIRFLOW_CTX_TASK_ID='read_s3_and_add_size_color' AIRFLOW_CTX_EXECUTION_DATE='2024-07-19T08:13:53.191214+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-19T08:13:53.191214+00:00'
[2024-07-19T08:17:41.922+0000] {base.py:83} INFO - Using connection ID 'aws_s3' for task execution.
[2024-07-19T08:17:41.923+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-07-19T08:17:59.509+0000] {logging_mixin.py:188} INFO - Visited: https://www.musinsa.com/app/goods/2612829
[2024-07-19T08:18:01.100+0000] {logging_mixin.py:188} INFO - Clicked button at div[35] on https://www.musinsa.com/app/goods/2612829
[2024-07-19T08:18:01.347+0000] {logging_mixin.py:188} INFO - Clicked button 1 at div[37] on https://www.musinsa.com/app/goods/2612829
[2024-07-19T08:18:01.972+0000] {logging_mixin.py:188} INFO - Clicked button 2 at div[37] on https://www.musinsa.com/app/goods/2612829
[2024-07-19T08:18:02.099+0000] {logging_mixin.py:188} INFO - test_option_1 for https://www.musinsa.com/app/goods/2612829: ['[베이직 ver.] 블랙', '[베이직 ver.] 크림(품절)', '[베이직 ver.] 그린(품절)', '[베이직 ver.] 인디핑크(품절)', '[베이직 ver.] 멜란지그레이(품절)', '[베이직 ver.] 마젠타(품절)', '[베이직 ver.] 네이비', '[베이직 ver.] 화이트', '[링거 ver.] 크림(품절)', '[링거 VER.] 크림(품절)', '[링거 ver.] 네이비(마지막 1개)', '[링거 VER.] 네이비(품절)']
[2024-07-19T08:18:02.100+0000] {logging_mixin.py:188} INFO - test_option_2 for https://www.musinsa.com/app/goods/2612829: ['S', 'M', 'L', 'XL']
[2024-07-19T08:18:25.700+0000] {logging_mixin.py:188} INFO - Visited: https://www.musinsa.com/app/goods/1191407
[2024-07-19T08:18:27.273+0000] {logging_mixin.py:188} INFO - Clicked button at div[36] on https://www.musinsa.com/app/goods/1191407
[2024-07-19T08:18:27.389+0000] {logging_mixin.py:188} INFO - Clicked button 1 at div[38] on https://www.musinsa.com/app/goods/1191407
[2024-07-19T08:18:27.770+0000] {logging_mixin.py:188} INFO - Clicked button at div[38] on https://www.musinsa.com/app/goods/1191407
[2024-07-19T08:18:27.835+0000] {logging_mixin.py:188} INFO - test_option_1 for https://www.musinsa.com/app/goods/1191407: ['S', 'M', 'L']
[2024-07-19T08:18:27.835+0000] {logging_mixin.py:188} INFO - test_option_2 for https://www.musinsa.com/app/goods/1191407: ['0']
[2024-07-19T08:23:27.903+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/code_test/airflow_size_color.py", line 115, in read_s3_and_add_size_color
    size_options, color_options = visit_websites(description_list)
  File "/opt/airflow/dags/code_test/airflow_size_color.py", line 130, in visit_websites
    sizes, colors = visit_website([sub_list])
  File "/opt/airflow/dags/code_test/airflow_size_color.py", line 24, in visit_website
    driver = webdriver.Remote(command_executor=remote_webdriver, options=options)
  File "/home/airflow/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 212, in __init__
    self.start_session(capabilities)
  File "/home/airflow/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 299, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
  File "/home/airflow/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "/home/airflow/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: Could not start a new session. New session request timed out 
Host info: host: '6be934e86a63', ip: '172.19.0.4'
Build info: version: '4.22.0', revision: 'c5f3146703'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.146.1-microsoft-standard-WSL2', java.version: '17.0.11'
Driver info: driver.version: unknown
Stacktrace:
    at org.openqa.selenium.grid.sessionqueue.local.LocalNewSessionQueue.addToQueue (LocalNewSessionQueue.java:221)
    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.lambda$new$0 (NewSessionQueue.java:68)
    at org.openqa.selenium.remote.http.Route$TemplatizedRoute.handle (Route.java:192)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:360)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.execute (NewSessionQueue.java:128)
    at org.openqa.selenium.remote.tracing.SpanWrappedHttpHandler.execute (SpanWrappedHttpHandler.java:87)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:360)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.grid.router.Router.execute (Router.java:87)
    at org.openqa.selenium.grid.web.EnsureSpecCompliantResponseHeaders.lambda$apply$0 (EnsureSpecCompliantResponseHeaders.java:34)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:360)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.remote.http.Route$NestedRoute.handle (Route.java:270)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:360)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.remote.AddWebDriverSpecHeaders.lambda$apply$0 (AddWebDriverSpecHeaders.java:35)
    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)
    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)
    at org.openqa.selenium.netty.server.SeleniumHandler.lambda$channelRead0$0 (SeleniumHandler.java:44)
    at java.util.concurrent.Executors$RunnableAdapter.call (None:-1)
    at java.util.concurrent.FutureTask.run (None:-1)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (None:-1)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (None:-1)
    at java.lang.Thread.run (None:-1)
[2024-07-19T08:23:27.933+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=crawling_dag_v2, task_id=read_s3_and_add_size_color, execution_date=20240719T081353, start_date=20240719T081741, end_date=20240719T082327
[2024-07-19T08:23:27.958+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 460 for task read_s3_and_add_size_color (Message: Could not start a new session. New session request timed out 
Host info: host: '6be934e86a63', ip: '172.19.0.4'
Build info: version: '4.22.0', revision: 'c5f3146703'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.146.1-microsoft-standard-WSL2', java.version: '17.0.11'
Driver info: driver.version: unknown
Stacktrace:
    at org.openqa.selenium.grid.sessionqueue.local.LocalNewSessionQueue.addToQueue (LocalNewSessionQueue.java:221)
    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.lambda$new$0 (NewSessionQueue.java:68)
    at org.openqa.selenium.remote.http.Route$TemplatizedRoute.handle (Route.java:192)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:360)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.execute (NewSessionQueue.java:128)
    at org.openqa.selenium.remote.tracing.SpanWrappedHttpHandler.execute (SpanWrappedHttpHandler.java:87)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:360)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.grid.router.Router.execute (Router.java:87)
    at org.openqa.selenium.grid.web.EnsureSpecCompliantResponseHeaders.lambda$apply$0 (EnsureSpecCompliantResponseHeaders.java:34)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:360)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.remote.http.Route$NestedRoute.handle (Route.java:270)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:360)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)
    at org.openqa.selenium.remote.AddWebDriverSpecHeaders.lambda$apply$0 (AddWebDriverSpecHeaders.java:35)
    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)
    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)
    at org.openqa.selenium.netty.server.SeleniumHandler.lambda$channelRead0$0 (SeleniumHandler.java:44)
    at java.util.concurrent.Executors$RunnableAdapter.call (None:-1)
    at java.util.concurrent.FutureTask.run (None:-1)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (None:-1)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (None:-1)
    at java.lang.Thread.run (None:-1); 1028)
[2024-07-19T08:23:28.004+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-19T08:23:28.050+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
